contracts/cw-proposal-multiple/src/tests.rs

```diff
@@ -3534,6 +3562,7 @@ fn test_active_threshold_none() {
     let max_voting_period = cw_utils::Duration::Height(6);
     let instantiate = InstantiateMsg {
         min_voting_period: None,
+        close_proposal_on_execution_failure: true,
```

Comment:
Would be great to add a test for this to this file / module. Know it's pretty much the same code as cw-proposal-single, but it'll help prevent future regressions.